sds_data_manager/constructs/instrument_lambdas.py

```diff
@@ -37,7 +37,7 @@ def __init__(
             Path to the Lambda code directory
         instrument_target : str
             Target data product (i.e. expected product)
-        instrument_sources : str
+        instrument_sources : list
```

Comment:
should this type be `list[str]`? also what does this look like, is it a S3 path? Can you add that to the docstring here?